R/db.R

```diff
@@ -168,7 +169,16 @@ write_data_safely <- function(dbconn, tblname, df, key) {
       raw_names <- DBI::dbGetQuery(dbconn, 'SELECT raw_name FROM column_names')[[1]]
       column_map <- column_map[!is.element(column_map$raw_name, raw_names), ]
       if (NROW(column_map) > 0) {
-        dbWriteTable(dbconn, 'column_names', column_map, append = TRUE, row.names = FALSE)
+        if (isTRUE(safe_columns)) {
+          stop("Safe Columns Error: Your function call is adding additional ",
+            "columns to a cache that already has pre-existing columns. This ",
+            "would suggest your cache is invalid and you should wipe the cache ",
+            "and start over.")
```

Comment:
Can you print what columns in the error message and save them to a global variable?